configuration/forms/jsonforms/HMIS-ACP-039-HIVSelfTestingRegister.json

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
Form Encounter type d325a161-bc6a-4d62-aec3-80cbb555778a could not be found. Please ensure that this encountertype is either loaded by Iniz or loaded in the system before Iniz runs.

Reply:
@Daphne210 
When you check this class where encounterTypes should exist this encounter type is not defined 
https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/master/api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java